vispy/app/backends/_tk.py

```diff
@@ -0,0 +1,701 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+vispy backend for Tkinter.
+"""
+
+from __future__ import division
+
+from time import sleep
+
+from ..base import (BaseApplicationBackend, BaseCanvasBackend,
+                    BaseTimerBackend)
+from ...util import keys
+from ...util.ptime import time
+
+
+# -------------------------------------------------------------------- init ---
+
+# Import
+try:
+    import sys
+    import os
+
+    _tk_on_linux, _tk_on_darwin, _tk_on_windows = \
+        map(sys.platform.startswith, ("linux", "darwin", "win"))
+    _tk_pyopengltk_imported = False
+
+    import tkinter as tk
+    from OpenGL import GL
```

Comment:
This should use `gloo` instead with a nested import, e.g.:

https://github.com/vispy/vispy/blob/master/vispy/app/backends/_qt.py#L683

and then use `gloo` GL methods/constants/whatever

Reply:
The backend OpenGL frame, `pyopengltk` explicitly uses the `OpenGL` module, so I also include it here (it's going to be included anyway). I only use it to clear the canvas initially (`GL.glClear` and `GL.glClearColor`).

Reply:
The backend OpenGL frame, `pyopengltk` explicitly uses the `OpenGL` module, so I also include it here (it's going to be included anyway). I only use it to clear the canvas initially (`GL.glClear` and `GL.glClearColor`).

Reply:
Can you add a comment along these lines so that the next person who looks at the code understands why it's done this way?

Reply:
Ok